test/test_duckdb.py

```diff
@@ -0,0 +1,120 @@
+from sql_metadata import Parser
+
+
+def test_pivot_operator():
+    source = """
+SELECT *
+  FROM monthly_sales
+    PIVOT(SUM(amount) FOR MONTH IN ('JAN', 'FEB', 'MAR', 'APR'))
+      AS p
+  ORDER BY EMPID
+    """
+    parser = Parser(source)
+    assert ["monthly_sales"] == parser.tables
+
+def test_table_named_pivot():
+    parser = Parser("from pivot")
+    assert parser.tables == ["pivot"]
+    assert parser.query_type == "SELECT"
+    parser = Parser("select * from pivot join other using (id)")
+    assert "pivot" in parser.tables and "other" in parser.tables
```

Comment:
this is unfortunately getting confused by the new handling of `pivot` and as a result interprets `join` as a potential table name as well.

Reply:
Seems fine, hopefully a pretty rare case!